docs/operator.md

```diff
@@ -131,6 +132,41 @@ A strategy the Operator uses for [upgrades](update.md#more-on-upgrade-strategies
 | ----------- | ---------- |
 | :material-code-string: string     | `SmartUpdate`              | 
 
+## <a name="operator-unsafeflags-section"></a>Unsafe flags section
+
+The `unsafeFlags` section in the [deploy/cr.yaml  :octicons-link-external-16:](https://github.com/percona/percona-xtradb-cluster-operator/blob/main/deploy/cr.yaml) file contains various configuration options to prevent users from configuring a cluster with unsafe parameters.
+
+### `unsafeFlags.tls`
+
+Prevents users from configuring a cluster without TLS/SSL certificates (if `false`, the Operator will automatically change unsafe parameters to safe defaults).
+
+| Value type  | Example    |
+| ----------- | ---------- |
+| :material-toggle-switch-outline: boolean     |`false` |
+
+### `unsafeFlags.pxcSize`
+
+Prevents users from configuring a cluster with unsafe parameters: starting it with less than 3 Percona XtraDB Cluster instances (if `false`, the Operator will automatically change unsafe parameters to safe defaults).
+
+| Value type  | Example    |
+| ----------- | ---------- |
+| :material-toggle-switch-outline: boolean     |`false` |
+
+### `unsafeFlags.proxySize`
+
+Prevents users from configuring a cluster with less than 2 ProxySQL or HAProxy Pods, (if `false`, the Operator will automatically change unsafe parameters to safe defaults).
+
+| Value type  | Example    |
+| ----------- | ---------- |
+| :material-toggle-switch-outline: boolean     |`false` |
+
+### `unsafeFlags.backupIfUnhealthy`
+
+Prevents running backup on a cluster with [failed health checks :octicons-link-external-16:](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes).
```

Comment:
it's not just about failed health checks, it allows users to run backups even if cluster status is not `ready`